test/auro-flight-main.test.js

```diff
@@ -5,21 +5,21 @@ describe('auro-flight-main', () => {
 
   it('auro-flight-main is accessible', async () => {
     const el = await fixture(html`
-      <auro-flight-main departureTime="2022-05-04T00:30:00-07:00" departureStation="SEA" arrivalTime="2022-05-04T11:55:00-04:00" arrivalStation="PVD" flights='["AS 1436"]'></auro-flight-main>
+      <auro-flight-main departureTime="2022-05-04T00:30:00-07:00" departureStation="SEA" arrivalTime="2022-05-04T11:55:00-04:00" arrivalStation="PVD" flights='["AS 1436"]' reroutedArrivalStation={null} reroutedDepartureStation={null}></auro-flight-main>
```

Comment:
What is the need for the `null` statements in the test?

Reply:
Not needed, has been removed.